runtime/libpgmath/lib/common/CMakeLists.txt

```diff
@@ -157,11 +176,9 @@ else() # Generic Architecture
     mth_128defs_init.c
     mth_128defs_stats.c)
   libmath_add_object_library("${MTH_INTRINSICS_SRCS}" "${FLAGS}" "${DEFINITIONS}" "mth_intrinsics")
- 
+
   set(SRCS
-    kidnnt.c
-    sincos.c
-    sincosf.c)
+    kidnnt.c)
```

Comment:
Why is this change necessary?

Reply:
It was for debugging purposes only, thanks for pointing it out, reverting those deletions asap.